sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeBeforeDownloadTest.java

```diff
@@ -928,6 +930,103 @@ public ServiceCall answer(InvocationOnMock invocation) throws Throwable {
         PreferencesStorage.remove(PREFERENCE_KEY_DOWNLOADED_RELEASE_ID);
     }
 
+    @Test
+    public void shouldChangeDistributionGroupIdIfStoredIdDoesntMatchDownloadedId() throws Exception {
```

Comment:
We are verifying this behavior by mocking a previous download, so this behavior is right after start and before an api call is made, it should be in another test class if possible (like before api).

Or this could be in a new class.

Reply:
Thanks for clarification. Moved to `DistributeBeforeApiSuccessTest` class.